### DISCUSSION TITLE
FL Sunseeker LEDs - how to enable?

### DISCUSSION THREAD

**Discussion by johnny2678** [tagyoureit/nodejs-poolController#475] [App and O/S Config Questions] [answered]
Saw support was added for these with 7.7.0 but not sure how to enable?  OCP still has my LEDs registered as Intellibrites.  

Can I use njspc + FL Sunseeker lights with an OCP?

![image](https://user-images.githubusercontent.com/8219743/172185412-847fef48-9192-47df-95ca-9a33f8aa6be1.png)

**Comment by rstrouse** [comment, marked as answer]
Unfortunately, you cannot use the EasyTouch controller to control those and have the colors match up.  It's not even close.  You can however, use an external relay to match with the color list.  However, it is a list and we didn't get enough participation to create the gradient graphics for all the themes.

Every person responsible for propagating such a half-baked idea as the light controls used by all the manufacturers should be questioned about every idea they ever had.  They are probably the source of tube socks, shorts, and Birkenstocks.  The ones that think they made it better by coming up with more, rearranged clicks are probably responsible for the fish milkshake.

**Reply by johnny2678** [reply]
Ha! no worries.  Just wanted to make sure I wasn't missing out.  I rarely change colors anyways but wanted to give it a shot when I saw it in the feature list.  Easy enough to toggle on/off to get to the right color, but yea, what a crazy light controller concept.
